edx/edx-ora2
# [BD-05] [EDUCATOR-5478] [BB-3613] Use webpack to compile sass assets as well

This PR gets rid of the existing python-based system for compiling SCSS in favour of a webpack based mechanism for js and scss. 

JIRA: [EDUCATOR-5478](https://openedx.atlassian.net/browse/EDUCATOR-5478)

**What changed?**

- This PR changes the webpack config to also compile the sass files
- It also updates the webpack config to output to a new directory instead of placing output files in the same dir. 
- It updates the python code to find the compiled assets in the new place
- It removes the old python-based scripts for compiling sass

**Developer Checklist**


**Testing Instructions**

- Install the PR version of the block
- Make sure that the block still works without issues. 

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
